deps/openssl/doc/UPGRADING.md

```diff
@@ -280,6 +280,13 @@ and the other is the older one. sections 6.1 and 6.2 describe the two
 types of files. Section 6.3 explains the steps to update the files.
 In the case of upgrading 1.0.2f there were no changes to the asm files.
 
+Files changed between two tags can be manually inspected using:
+```
+https://github.com/openssl/openssl/compare/OpenSSL_1_0_2k...OpenSSL_1_0_2l#files_bucket
```

Comment:
This example URL is inconsistent with the rest of this doc; the very first sentence at the beginning of this doc is:
> This document describes the procedure to upgrade openssl from 1.0.2e to 1.0.2f in Node.js.